### ISSUE TITLE
Can't run app from terminal [Ubuntu]

### ISSUE THREAD

**Issue by akashnimare** [electron-userland/electron-builder#2689]
electron-builder - `v19.53.6`

This used to work but I just noticed that after installing the app using the deb installer, I can't run the app from the command line.

Generated desktop entry 
```sh
[Desktop Entry]
Name=Zulip
Comment=Zulip Desktop Client for Linux
Exec="/opt/Zulip/zulip" %U
Terminal=false
Type=Application
Icon=zulip
Categories=Chat;GNOME;GTK;Network;InstantMessaging;
```

**Comment by develar** [comment]
I suggest you to stop providing `deb`, consider to use `snap`.

> I can't run the app from the command line.

Do you mean that `zulip` is not available? But we never add installer app bin to `PATH`.

**Comment by akashnimare** [comment]
> I suggest you to stop providing deb

I wish I could do that. There are some users who only use the deb installer and don't want to switch to snap or AppImage but we trying hard to encourage them to drop the deb. We do support AppImage and few people migrated to this format. We're also planning to push the `snap` format asap.

Back to the original issue, I think this used to work in `v19.13.0`.  The exec command `/opt/Zulip/zulip` starts the app successfully but not `zulip`.

**Comment by develar** [comment]
Duplicates #2682

**Comment by akashnimare** [comment]
@develar I fixed this issue in https://github.com/zulip/zulip-electron/commit/3eb4cf4f64106e2ecdd1612d0588528aa958e552. It used to work before we added the `afterInstall` script. The thing is the default `afterInstall` template links the app to the bin. But when we use a custom script the default script doesn't trigger. To make it work I added the content of default `afterInstall` to the custom script we added.
